crates/eframe/src/native/run.rs

```diff
@@ -223,6 +223,27 @@ fn run_and_exit(
     })
 }
 
+fn centere_window_pos(
+    monitor: Option<winit::monitor::MonitorHandle>,
+    native_options: &mut epi::NativeOptions,
+) {
+    // Get the current_monitor.
+    if let Some(monitor) = monitor {
+        let monitor_size = monitor.size();
+        let inner_size = native_options
+            .initial_window_size
+            .unwrap_or(egui::Vec2 { x: 800.0, y: 600.0 });
```

Comment:
This seems like a rather random assumption.

Perhaps `NativeOption::initial_window_size` shouldn't be an `Option`, and instead be a `Vec2` with a default of 800x600?

Reply:
@emilk yup, i also think so.

Reply:
@emilk  not sure about that though, it turns out breaking change on a lot of  `egui/examples` and there's also `min_window_size` /`max_window_size` on `NativeOptions` which is `Option<Vec2>`